docs/spelling-wordlist.txt

```diff
@@ -46,10 +48,37 @@ validator
 validators
 versioned
 schemas
+sortable
 
 Zac
 HD
 
 Berman
 Libera
 GPL
+
+# json properties
+additionalProperties
```

Comment:
I assume this is somehow autodoc pulling things in again? Basically none of these should really be here, because none of them are really words, they should always either be surrounded in backticks (which will disable spell-checking them) or be used in `:kw:`foo`` roles. But autodoc can be funny there and find these words in documents we don't control...

Reply:
I can try one more time to fix this 😄 

I have an idea that might work, but I can't promise much. It is indeed `autodoc` and `autoapi` pulling more stuff in.